packages/core-ethereum/src/indexer/index.mock.ts

```diff
@@ -0,0 +1,239 @@
+import EventEmitter from 'events'
```

Comment:
This is a great setup, but has lots of implicit knowledge which is undocumented. Reading through it one wonders, why have certain values/setups been chosen.

Reply:
Eg.?